packages/demo/src/AppCCP.svelte

```diff
@@ -278,14 +278,20 @@
 </main>
 
 <footer>
-    <a class="user-agreement" href="http">Nutzervereinbarung</a>
-    <a class="email" href="mailto:CCP@dkfz.de">CCP@dkfz.de</a>
-    <div class="copyright">
-        <span>&#169; 2023</span>
-        <a href="https://dktk.dkfz.de/en/clinical-platform/about-ccp"
-            >Clinical Comunication Platform (CCP)</a
-        >
-    </div>
+    <a
+        class="ccp"
+        href="https://dktk.dkfz.de/klinische-plattformen/ueber-die-ccp/about-ccp"
+    >
+        Clinical Communication Platform (CCP)
+    </a>
+    <a class="email" href="mailto:CCP@dkfz.de">Kontakt</a>
+    <a class="user-agreement" href="http" download="nutzervereinbarung"
+        >Nutzungsvereinbarung</a
+    >
+    <a class="privacy-policy" href="http" download="datenschutzerklaerung"
+        >Datenschutz</a
+    >
+    <a class="imprint" href="http">Impressum</a>
```

Comment:
Link should be to https://www.dkfz.de/de/impressum.html

Reply:
fixed

Reply:
unfortunately not, the "impress" link is still not working. I think you put it in the wrong a-tag

Reply:
yes, just was the wrong tag. the file downloads will need to wait till the office has them ready, as we discussed.